### DISCUSSION TITLE
Need 120.2 47.2.6

### DISCUSSION THREAD

**Discussion by Zhou-Shilin** [IzzelAliz/Arclight#1142] [Ideas]
Current forge version of arclight 1.20.1 is toooo low (47.1.1 40+versions behind), and the lastest forge version is 47.2.6
AE2 mod needs higher versions

**Comment by wrldmap** [comment]
Official builds go up to 47.2.0 without modification. [Download that here.](https://github.com/IzzelAliz/Arclight/suites/17495389974/artifacts/999613688)
You can replace a file in this copy to force it to install the newest Forge version. You will need a copy of Forge 47.2.6 Server Software, and the newest version of Arclight. (Linked above.) Run these until it stops for you to accept the EULA. This will be when you need the official Forge copy. Navigate to `[your arclight server]/libraries/net/minecraftforge/forge/1.20.1-47.x.x/` and` [your forge server]/libraries/net/minecraftforge/forge/1.20.1-47.2.6/`. There will be a `win_args.txt` and a `unix_args.txt` Replace these two on the Arclight server with the ones from the Forge server. Then, you can delete the Forge server, then start Arclight. Hope this helps!

**Reply by Zhou-Shilin** [reply]
Thank you very much!!
I tried to change forge version by replacing before i opened this discussion, but it crashes. So I'll download the official build you mentioned.

**Reply by wrldmap** [reply]
Yeah, the replacing the forge version thing crashed for me too. I have no idea why, maybe its a problem with this build.
